cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/TestCommand.java

```diff
@@ -156,10 +171,18 @@ public void execute() {
             testReport = false;
         }
         BuildOptions buildOptions = constructBuildOptions();
+
         boolean isSingleFile = false;
         if (FileUtils.hasExtension(this.projectPath)) {
             try {
+                if (buildOptions.dumpBuildTime()) {
```

Comment:
Let's take this out of the if condition and avoid repeating the code in both blocks